extensions/amp-dailymotion/dailymotion-api.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {addParamsToUrl} from '../../src/url';
+import {dict} from '#core/types/object';
+import {isAutoplaySupported} from '#core/dom/video';
```

Comment:
Looking at `vimeo-api.js`, it looks like only 0.1 checks `isAutoplaySupported` whereas 1.0 does not. Likely this is because it relies on `win` to be defined, which it won't be on Preact's first render.